docs/01-Overview.md

```diff
@@ -1,5 +1,5 @@
 ---
-id: introduction
+id: overview
```

Comment:
Would prefer to keep filenames lowercase unless there's a good reason to do otherwise

Reply:
Done. Will set  the sidebar label through meta-data.